src/lib/OpenEXRCore/chunk.c

```diff
@@ -1753,7 +1757,7 @@ exr_write_scanline_chunk_info (
     cinfo->data_offset              = 0;
     cinfo->packed_size              = 0;
     cinfo->unpacked_size =
-        compute_chunk_unpack_size (y, cinfo->width, cinfo->height, lpc, part);
+        compute_chunk_unpack_size (dw.min.x, y, cinfo->width, cinfo->height, lpc, part);
```

Comment:
now that dw.min.x is accounted for, I'm wondering if tests are exposing correctness with non-zero x dw?

Seeing this diff, I think I know why I was so confused trying to validate non-zero data windows in nanoexr....!

Reply:
yes, there are many tests with non-0 x data windows - the math was correct because of constraints / validation checks, but have mostly done this for consistency (and in case we want to change how it works in the future for different compression styles while I was in there, been thinking a lot about faster ways of loading sub-windows of stuff onto gpus and such)